docs/_tutorials/zero-one-adam.md

```diff
@@ -0,0 +1,131 @@
+
+```shell
+mpirun -np [num processes] -ppn [num GPUs on each node] -hostfile [hostfile] [MPI flags] python [training_script.py]
+```
+
+### 1.3 0/1 Adam Algorithm
+
+The detailed description of the 0/1 Adam algorithm can be seen from our [paper](https://arxiv.org/abs/2202.06009).
+
+### 1.4 Configuration of 0/1 Adam
+The 0/1 Adam feature can be used by setting the optimizer configuration options as follows. An example json config file is shown below.
+
+```json
+{
+  "train_batch_size": 4096,
+  "train_micro_batch_size_per_gpu": 16,
+  "optimizer": {
+    "type": "ZeroOneAdam",
+    "params": {
+      "lr": 1e-3,
+      "weight_decay": 0.01,
+      "bias_correction": false,
+      "var_freeze_step": 1000,
+      "var_update_scaler": 16,
+      "local_step_scaler": 1000,
+      "local_step_clipper": 16,
+      "cuda_aware": false,
+      "comm_backend_name": "nccl"
+    }
+  },
+  "gradient_clipping": 1.0,
+  "fp16": {
+    "enabled": true,
+    "loss_scale": 0,
+    "initial_scale_power": 16
+  }
+}
+```
+Please note the new parameters `var_freeze_step`, `var_update_scaler`, `local_step_scaler`, `local_step_clipper`, `cuda_aware` and `comm_backend_name` that have been added to support the 0/1 Adam feature:
+
+`var_freeze_step` is the latest step to update the variance. Using the notation from [0/1 Adam paper](https://arxiv.org/abs/2202.06009), it denotes the $\max\{i|i \in \mathcal{T}_v\}$. Note that this is different from the `freeze_step` in 1-bit Adam. The `var_freeze_step` is usually the end of the learning rate warmup and thus does not require tuning. Note that this hyperparameter is optional. In practice, we can avoid tuning this parameter by setting it to a very large number (larger than the total number of steps). Following this, 0/1 Adam still enjoys the non-trivial communication reduction without affecting the convergence speed.
+
+`var_update_scaler` is the interval to update the variance. Note that the update policy for variance follows an exponential rule, and `var_update_scaler` denotes the $\kappa$ factor (please refer to the [0/1 Adam paper](https://arxiv.org/abs/2202.06009) for detailed expression). In practice, we found its default value (16) is able to work well on most of the tasks, including BERT-Base/Large pretraining, GPT pretraining, and ImageNet training.
+
+`local_step_scaler` and `local_step_clipper` are two hyperparameters for learning rate based local step policy in 0/1 Adam. Informally, `local_step_scaler` denotes the number of steps 0/1 Adam takes before increasing its local step interval. `local_step_clipper` is the largest local step interval, i.e., the maximal number of communication rounds 0/1 Adam can skip before the next synchronization. The learning rate policy is the default policy used in 0/1 Adam. We can also trivially construct other policies by setting these two hyperparameters such as constant local step interval policy by setting `local_step_scaler=1` and `local_step_clipper=constant`.
```

Comment:
I believe `local_step_scaler` denotes the number of steps 0/1 Adam takes before multiplying its local step interval by 2? If so then it might be better to explicitly write so instead of "increasing its local step interval" which is a bit vague. Similary, we might want to describe explicitly how we increase the variance update interval based on var_update_scaler (I believe it's also multiplying by 2?).